go.mod

```diff
@@ -45,6 +45,7 @@ replace (
 )
 
 require (
+	github.com/Jeffail/gabs/v2 v2.6.1
```

Comment:
Separate the `vendor` folder change with `go.mod` change into the first commit, so it won't break the build if uses `git bisect`.